modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/parcellationmeshgeneration/v0.1.0.json

```diff
@@ -0,0 +1,43 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/atlas/parcellationmeshgeneration/v0.1.0/shapes/"
+    }
+  ],
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.3"
+  ],
+  "@type": "nxv:Schema",
+  "shapes": [
+    {
+      "@id": "this:ParcellationMeshGenerationShape",
+      "@type": "sh:NodeShape",
+      "label": "Parcellation mesh generation activity description.",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "targetClass": "nsg:ParcellationMeshGeneration",
+      "and": [
+        {
+          "@id": "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0/shapes/ActivityShape"
```

Comment:
I would suggest to use "node" instead of "@id" and neurosciencegraph/commons/activity/v0.1.4 instead of neurosciencegraph/commons/activity/v0.1.0